docs/templates/error_codes.md.j2

```diff
@@ -0,0 +1,7 @@
+| Category | Code |     | Message |
+| -------- | ---- | --- |-------- |
+{% for error_type, errors in error_types.items() %}
+{% for error in errors %}
+| {{ error_type }} <td colspan=2> {{ error['code'] }} <td colspan=3> {{ error['message'] }}|
+{% endfor %}
+{% endfor %}
```

Comment:
Please, automate adding empty lines at the end to avoid having these warning on GitHub :D

Reply:
done